docs/data/material/components/buttons/LoadingButtonsTransition.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import LoadingButton from '@mui/lab/LoadingButton';
+import Button from '@mui/material/Button';
```

Comment:
I think a `<Button loading>` is better than having `<Button>` and `<LoadingButton>`. I guess 9/10 projects would need a loading button so it's fine to bundle `CircularProgress` to the `Button`.

Reply:
Yes, this was discussed. Having a `loading` prop in Button is universally established and I think it's preferable to merge `LoadingButton` and `Button` to reduce API surface and confusion (why having 2 components with almost the same exact API?), even if it means an increase in the bundle size.

Q: Is there a way we could dynamically import the progress component that would be compatible with all bundlers? Are dynamic imports universally supported?

> Personally, as a user, I wouldn't want to use the Material UI Button if it loads a progress component that is different from the one that I want to use because it's bundle size bloat.

This also happens with component styles. Users load our components, which have all styles built-in and are included in the bundle, to later override them.